it/src/main/java/org/eclipse/jkube/integrationtests/jupiter/api/extension/GradleExtension.java

```diff
@@ -70,7 +72,8 @@ private JKubeGradleRunner getJKubeGradleRunner(ExtensionContext context) throws
       }
       var projectPath = rootPath.resolve("projects-to-be-tested").resolve("gradle");
       var gradleRunner = GradleRunner.create()
-        .withGradleInstallation(getGradleInstallation())
+        .withGradleDistribution(new URI("https://services.gradle.org/distributions/gradle-7.6-bin.zip"))
+//        .withGradleInstallation(getGradleInstallation())
```

Comment:
`getGradleInstallation` is no longer used anywhere. We should remove it.

Reply:
we yet need to see if we want to change this approach in a per-test basis. Left there for evaluation of options.